OctoAwesome/OctoAwesome.Client/OctoAwesome.Client.csproj

```
@@ -41,7 +41,7 @@
     <PackageReference Include="engenious.UI" Version="0.5.1.1-alpha" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="OpenAL.Soft" Version="1.19.1" Condition="$(RuntimeIdentifier.StartsWith('win')) Or ('$(RuntimeIdentifier)' == '' AND $([MSBuild]::IsOSPlatform('Windows')))"/>
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+	<PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
```

```suggestion
    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
```
Use space instead of tab